softdev.bib

```diff
@@ -2096,3 +2096,37 @@ @inproceedings{blackburn2004myths
     pages = {25--36},
     publisher = {ACM},
 }
+
+@article{cheney1970nonrecursive,
+    author = {Cheney, C. J.},
+    title = {A Nonrecursive List Compacting Algorithm},
+    journal = {Commun.~ACM},
+    volume = {13},
+    number = {11},
+    month = Nov,
+    year = {1970},
+    pages = {677--678},
+    publisher = {ACM},
+}
+
+@inproceedings{shahriyar2012down,
+    author = {Shahriyar, Rifat and Blackburn, Stephen M. and Frampton, Daniel},
+    title = {Down for the Count?  Getting Reference Counting Back in the Ring},
+    booktitle = {ISMM},
+    series = {ISMM '12},
+    year = {2012},
+    pages = {73--84},
+    month = Jun,
+    publisher = {ACM},
+}
+
+@article{dijkstra1978on,
```

Comment:
I suspect ol' Edsger wrote a lot of articles starting "on". Maybe `dijkstra1978gc` or something like that?

Reply:
Fair enough. Updated